config/locales/en.yml

```diff
@@ -129,3 +132,4 @@ en:
   errors:
     messages:
       improbable_phone: 'Mobile number is invalid. Please make sure you enter a 10-digit phone number.'
+      invalid_confirmation_code: 'Invalid confirmation code.'
```

Comment:
With the current site style, I prefer the shorter flash messages, otherwise they take up more room on the screen than the actual content.